README.md

```diff
@@ -110,7 +110,7 @@ you can use the following in your init.el:
 
 ## Clippy
 `rust-run-clippy` runs
-[Clippy](https://github.com/rust-lang/rust-clippy), a linter. 
+[Clippy](https://github.com/rust-lang/rust-clippy), a linter.
```

Comment:
Sorry about this unrelated whitespace change. I have a `'delete-trailing-whitespace` `'before-save-hook` which is responsible.

Reply:
No worries.